1_IGT_PP/Code/R/1_Analyses/fitting_orl_joint.Rmd

```diff
@@ -0,0 +1,203 @@
+---
+title: "Fitting Joint Test-Retest ORL"
+output: html_document
+date: "2023-03-07"
+---
+
+
+
+# -------------------------------------------
+# Setup
+## Load Packages
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+library(tidyverse)
+library(rstan)
+library(hBayesDM)
+library(bayesplot)
+library(here)
+library(miscTools)
+```
+
+
+
+## Load Data
+```{r}
+stan_data = readRDS(here("1_IGT_PP", "Data", "1_Preprocessed", "List_Sess1&2.rds"))
+full_data = readRDS(here("1_IGT_PP", "Data", "1_Preprocessed", "Dataframe_Sess1&2.rds"))
+```
+
+
+
+# -------------------------------------------
+# Fit Model
+```{r, eval = F}
+# Compile model
+orl_model = stan_model(here("1_IGT_PP", "Code", "Stan", "igt_orl_joint.stan"))
+
+
+# Fit model
+orl_fit = sampling(orl_model, 
+                   data   = stan_data, 
+                   iter   = 5000, 
+                   warmup = 1000,
+                   chains = 6, 
+                   cores  = 6,
+                   seed   = 43210)
```

Comment:
FYI we probably don't need this many samples. Generally, unless we are really interested in tail quantities, something around 5000 samples is good enough (and saves some space! ha). But more is never bad 🤓

Reply:
Oops, I meant to respond to this the first go around. That sounds good though. I can't remember why I did that but thinking about it some more, would it be better to refit it with fewer chains as in the other models? That is, to be consistent with how y'all have fit them in the past?

Reply:
@jeremy-haynes it is fine either way honestly! There is really no benefit in having a consistent number of samples across different applications, it really just comes down to whether the number of samples you have is enough to make inference on what you care about